server/channels/store/sqlstore/scheduled_post_store.go

```diff
@@ -206,6 +206,12 @@ func (s *SqlScheduledPostStore) PermanentlyDeleteScheduledPosts(scheduledPostIDs
 }
 
 func (s *SqlScheduledPostStore) UpdatedScheduledPost(scheduledPost *model.ScheduledPost) error {
+	scheduledPost.PreSave()
```

Comment:
This should be a `PreUpdate` method instead of a `PreSave`. I don't know if we actually have that naming convention documented anywhere since it's not super clear, but I think most places in our code only call `PreSave` when the object is first created

Reply:
We do actually have that convention. Done! Added and used `PreUpdate`.